### DISCUSSION TITLE
项目自荐

### DISCUSSION THREAD

**Discussion by krongk** [taowen/awesome-lowcode#50] [General]
https://www.baklib.com/

BakLib致力于低代码内容建设平台，能够快速搭建诸如文档类官网、帮助中心、FAQ、Guide、知识库、在线手册，并一键发布到网络上。

**Comment by foreverwang** [comment]
这个平台能做的 语雀都做了

**Reply by krongk** [reply]
跟语雀 的方向完全不一样，你可以理解Baklib是一个自助建站。

**Comment by yonghu86** [comment]
http://www.rdiframework.net/
RDIFramework.NET，基于全新.NET Framework与.NET Core的快速信息化系统敏捷开发、整合低代码平台，给用户和开发者最佳的.Net框架部署方案。为企业快速构建跨平台、企业级的应用提供了强大支持。

**Comment by qmdx** [comment]
Java 低代码组件库  爱组搭 文档： http://aizuda.com

开源仓库 GitHub： https://github.com/aizuda  
Gitee ：  https://gitee.com/aizuda

欢迎交流！！

**Comment by yangaijun** [comment]
自荐个给开发者使用的免费纯前端中后台类可视化开发：https://light2f.com

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬们有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/
